### DISCUSSION TITLE
Commercial support options

### DISCUSSION THREAD

**Discussion by nicholascar** [eclipse-rdf4j/rdf4j#4751] [General]
rdf4j is one of the candidate systems for operationally deployed RDF databases. Some organisations either require or just need commercial support for such installations.

What commercial support options are there for rdf4j? Are there companies that offer Service Level Agreements for rdf4j installatoions and continued operations?

**Comment by hmottestad** [comment]
My company HASMAC AS offers consulting for RDF4J focused on bug fixes, performance optimization and custom features. I haven't delved into ops yet, but it could be an interesting opportunity. 

For the time being I would recommend getting in contact with Ontotext. Their database "GraphDB" builds on top of RDF4J and they offer a number of options for operational support: https://www.ontotext.com/services/support-and-operations/
